lib/api/responses/blockchain_response.dart

```diff
@@ -16,29 +16,33 @@ class BlockchainData {
   String ticker;
   int precision;
   int blockchainFee;
+  String extraData;
 
   BlockchainData(
       {required this.name,
       required this.unit,
       required this.logo,
       required this.ticker,
       required this.precision,
-      required this.blockchainFee});
+      required this.blockchainFee,
+      required this.extraData,});
```

Comment:
Are we going to use blockchain.extradata in any other way than sending the genesis hash ?
If not, could we rename it to `xchGenesisHash` or `aggSigMeExtraData` instead?